css/css-values/support/mixed-units-12.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      body {
+        margin: 0;
+        font-size: 16px;
+      }
+      .box {
+        background: rgb(0, 0, 255);
+        width: 100%;
+        height: 10px;
+      }
+
+      @media (width: calc(90vw * 10px * 10px)) {
```

Comment:
Ditto

Reply:
Resolved in [2a682d8](https://github.com/web-platform-tests/wpt/pull/42517/commits/2a682d8b77a35036c0a71b538fda4730a963fc6d)